### ISSUE TITLE
SELinux label format error for docker v1.11+

### ISSUE THREAD

**Issue by feiskyer** [kubernetes/kubernetes#36136]
From docker's [documentation](https://docs.docker.com/engine/reference/run/#/security-configuration), selinux labels format has been changed to

```
--security-opt="label=user:USER"   : Set the label user for the container
--security-opt="label=role:ROLE"   : Set the label role for the container
--security-opt="label=type:TYPE"   : Set the label type for the container
--security-opt="label=level:LEVEL" : Set the label level for the container
--security-opt="label=disable"     : Turn off label confinement for the container
```

so we need to change `label:` to `label=` for docker 1.11+ (and keep `label:` for low version docker). I'd like to fix the issue in the following PR after #34811.   

cc/ @pmorie @timstclair @yujuhong

**Comment by yujuhong** [comment]
@feiskyer I think docker v1.11 would display warnings, but would still accept `:`. Is that not correct?
Of course we should still clean this up.

**Comment by feiskyer** [comment]
> @feiskyer I think docker v1.11 would display warnings, but would still accept :. Is that not correct?
Of course we should still clean this up.

Didn't dig into docker's code yet. Since we have passed all SELinux testing on Docker v1.12, I think it still accept `:`.  I think we still need to clean this up in case it drops such support in future releases.

cc/ @kubernetes/sig-node

**Comment by feiskyer** [comment]
Already fixed in #40903.
